metricflow/model/semantics/metric_lookup.py

```diff
@@ -127,6 +127,8 @@ def measures_for_metric(
                     column_association_resolver=column_association_resolver,
                 ).create_from_where_filter_intersection(input_measure.filter),
                 alias=input_measure.alias,
+                join_to_timespine=input_measure.join_to_timespine,
+                fill_nulls_with=input_measure.fill_nulls_with,
```

Comment:
These are new..... I'm guessing from another branch? Let's revert these from this PR before merge.

Reply:
dammit I didn't look at my PR after putting it up 😅  redo time